src/__tests__/components/dashboard/health-record/Labs.test.js

```diff
@@ -1,13 +1,47 @@
 import { fullRenderComponent } from '../../../../utils/testHelpers';
 import Labs from '../../../../components/dashboard/health-record/Labs';
+import * as mocks from '../../../../__mocks__/labMocks';
+import VerticalTimeline from '../../../../components/dashboard/shared/VerticalTimeline';
```

Comment:
One nitpick, I would change the import for Vertical Timeline and move it above the mocks so all the component imports are together.

Reply:
Travis is failing as well so that needs to be addressed